client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -0,0 +1,74 @@
+                        data-ng-repeat="shortcut in $ctrl.shortcuts track by $index"
+                        data-href="{{:: shortcut.href }}">
+                        <span data-translate="{{ 'user_account_dashboard_shortcut_' + shortcut.key | translate }}"></span>
+                    </oui-tile-button>
+                </oui-tile>
+            </div>
+
+            <div class="col mb-4 mx-2">
+                <oui-tile data-heading="{{:: 'user_account_dashboard_last_bill' | translate }}">
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_reference' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.billId }}">
+                    </oui-tile-definition>
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_date' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.date | date}}">
+                    </oui-tile-definition>
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_amount' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.priceWithTax.text }}">
+                    </oui-tile-definition>
+                    <a
+                        class="oui-button oui-button_secondary mx-auto my-5 d-block"
+                        data-ng-href="{{ $ctrl.lastBill.url }}"
+                        target="_blank"
+                        rel="noopener"
+                        data-translate="user_account_dashboard_last_bill_see">
+                    </a>
+                </oui-tile>
+            </div>
+
+            <div class="col mb-4 mx-2">
+                <oui-tile data-heading="{{:: 'user_account_dashboard_profile' | translate }}">
+                    <div class="row">
+                        <div class="col-md-3">
+                            <span class="oui-icon oui-icon-user_extra-thin"></span>
+                        </div>
+                        <div class="col-md-7">
+                            <div class="mb-4">
+                                <p>
+                                    <strong data-ng-bind=":: $ctrl.user.firstname + ' ' $ctrl.user.name"></strong>
+                                </p>
+                                <p data-ng-bind="::$ctrl.user.nichandle"></p>
+                            </div>
+                            <p data-ng-bind="::$ctrl.user.address"></p>
+                            <p data-ng-bind=":: $ctrl.user.zip + ' ' $ctrl.user.city"></p>
+                            <p data-ng-bind="::$ctrl.user.phone"></p>
+                            <p class="text-wrap" data-ng-bind="::$ctrl.user.email"></p>
+                            <p>
+                                <span data-translate="user_account_dashboard_profile_support_level"></span>
+                                <span data-translate="{{ 'user_account_support_level_section_' + $ctrl.supportLevel.level | translate }}"></span>
```

Comment:
```suggestion
                                <span data-translate="{{:: 'user_account_support_level_section_' + $ctrl.supportLevel.level }}"></span>
```
I don't think this works right now ?

Reply:
It actually does but will change it 😉